src/System.Runtime.InteropServices/src/System/Runtime/InteropServices/RuntimeEnvironment.cs

```diff
@@ -23,7 +23,10 @@ public static bool FromGlobalAccessCache(System.Reflection.Assembly a)
         }
         public static string GetRuntimeDirectory()
         {
-            return Path.GetDirectoryName(typeof(object).Assembly.Location) + Path.DirectorySeparatorChar;
```

Comment:
Conceptually, however, UAP is supposed to be isolated. As we don't use Configuration ourselves anymore I think it is best that we look locally for the machine.config.